lib/events.js

```diff
@@ -48,10 +48,19 @@ function off (state, eventName, handler) {
 /**
  * trigger a specified event
  *
- * @param  {String} eventName   Name of event
+ * @param  {String} eventName    Name of event
+ * @param  {...*} [options]      Options
  */
 function trigger (state, eventName) {
-  state.emitter.emit(eventName)
+  var emitter = state.emitter
+  var len = arguments.length
+  var args = Array(len - 1)
+
+  for (var i = 0; i < len - 1; i++) {
+    args[i] = arguments[i + 1]
+  }
```

Comment:
You can simplify the code above with this line

```js
var args = [].slice.call(arguments, 1)
```

Here is a minimal showcase

```js
(function () {
  var args = [].slice.call(arguments, 1)
  console.log(args)
})('eventName', 1, 2, 3)
```

Does that make sense?